compose/cli/main.py

```diff
@@ -100,11 +98,13 @@ class TopLevelCommand(Command):
       stop               Stop services
       up                 Create and start containers
       migrate-to-labels  Recreate containers to add labels
+      version            Show the Docker-Compose version information
 
     """
     def docopt_options(self):
         options = super(TopLevelCommand, self).docopt_options()
-        options['version'] = "docker-compose %s" % __version__
+        options['version'] = get_versions() \
+            + '\n\nThe Usage of --version is deprecated, use the command `version` instead.'
```

Comment:
I don't think this needs to be deprecated. `docker` supports both. The `--version` is just a short one-line version. I would leave this unchanged.